pkg/ui/console/client.go

```diff
@@ -778,6 +779,15 @@ func setAnalyticsProperties(props map[string]interface{}) {
 	})
 }
 
+func setUserId(databaseId string) {
```

Comment:
```suggestion
func setUserId(newId string) {
```
maybe? calling it databaseId leaks upstream implementation details into this fn, which innocently sets the id and moves on 😂